### PR TITLE
ADD: Sponsors section

### PR DESCRIPTION
**Pull request by aryanh0911**
initial commit for the Sponsors section in the home page.

### CODE REVIEW

File: src/Components/Sponsors/SponsorsStyle.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.container {
+    /* display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center; */
+/* 
+    margin-right: auto;
+    margin-left: auto; */
```
Comment (bj-jiwrajka) [inline src/Components/Sponsors/SponsorsStyle.css]: if the code is not required, remove it

File: src/Components/Sponsors/SponsorCards/SponsorCardsStyles.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  /* border: 2px red solid; */
+  margin: 0 auto;
+  margin-top: 2rem;
+
+}
+
+
+.spons_img {
+  height: 300px;
+  width: 280px;
```
Comment (bj-jiwrajka) [inline src/Components/Sponsors/SponsorCards/SponsorCardsStyles.css]: avoid using px, convert it to relative units such as rem/em

File: src/Components/Sponsors/SponsorCards/SponsorCards.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```
Comment (bj-jiwrajka) [inline src/Components/Sponsors/SponsorCards/SponsorCards.jsx]: import of react is not required after react version 17

**Review by bj-jiwrajka — CHANGES_REQUESTED** [review]

File: src/Components/Sponsors/SponsorHeading/SponsorHeading.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import './SponsorHeadingStyles.css'
+
+function SponsorHeading() {
+  return (
+    <div className='header_container'>
+       <img className='drum_left' src="https://res.cloudinary.com/dyn3q57b8/image/upload/f_auto,q_auto/v1/Oikyotaan%20'24%20assets/Sponsors/drumLeft" alt="a drum" />
+       <div className='title_div'>
+            <p className='title'>Sponsors</p>
```
Comment (bj-jiwrajka) [inline src/Components/Sponsors/SponsorHeading/SponsorHeading.jsx:8]: font doesn't match with the design and the letters are uppercase in the design

**Review by bj-jiwrajka — CHANGES_REQUESTED** [review]

File: src/Components/Sponsors/SponsorCards/SponsorCards.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    <div className='grid_container'>
+      {Cards.map((item,i)=>(
+        <div className="card" key={item.id}>
+          <img className='spons_img' src={item} alt="a sponsor" />
+        </div>
+      ))}
+    </div>
+  )
+}
+
```
Comment (bj-jiwrajka) [inline src/Components/Sponsors/SponsorCards/SponsorCards.jsx:15]: avoid using functional component

File: src/Components/Sponsors/SponsorCards/SponsorCardsStyles.css [hunk 6]
```diff
@@ -0,0 +1,42 @@
+.grid_container{
+  /* display: grid;
+  grid-template-columns: auto auto auto; */
```
Comment (bj-jiwrajka) [inline src/Components/Sponsors/SponsorCards/SponsorCardsStyles.css]: remove unnecessary comments

**Review by bj-jiwrajka — CHANGES_REQUESTED** [review]

**Comment by bj-jiwrajka** [comment]
Some more changes:

- Use module css (take reference from rest of the pages)
- Card size is too big, make it small

**Review by Hrishita-Paul — CHANGES_REQUESTED** [review]
1)Reduce the shadowing effect on the images
2)Add the images of the flowers in the background
3)Check the mobile view on the figma link and make changes

**Pull request closed without merging**